server/bin/pbench-index.py

```diff
@@ -124,7 +129,7 @@ def main(options, name):
         return 2
     except BadConfig as e:
         print(f"{name}: {e}", file=sys.stderr)
-        return 3
+        return BAD_CFG
     except JsonFileError as e:
         print(f"{name}: {e}", file=sys.stderr)
         return 8
```

Comment:
Thanks; but 0, 1, 2, and 8 are still returned as literals.

Reply:
Yes, let's get those literals changed as well.

Reply:
ok